pipelines/defaults.json

```diff
@@ -56,7 +56,8 @@
             "special.functional",
             "special.jck",
             "dev.openjdk",
-            "dev.system"
+            "dev.system",
+            "dev.jck"
```

Comment:
We should not add `dev.jck` into weeklyDefault. It should run on available platforms with NFS.